[2022-08-07 15:05:34,953] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: load_ad_data.load_briefing_data manual__2022-08-07T11:51:07.834855+00:00 [queued]>
[2022-08-07 15:05:35,005] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: load_ad_data.load_briefing_data manual__2022-08-07T11:51:07.834855+00:00 [queued]>
[2022-08-07 15:05:35,005] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-07 15:05:35,005] {taskinstance.py:1377} INFO - Starting attempt 5 of 6
[2022-08-07 15:05:35,005] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-07 15:05:35,014] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): load_briefing_data> on 2022-08-07 11:51:07.834855+00:00
[2022-08-07 15:05:35,016] {standard_task_runner.py:52} INFO - Started process 101473 to run task
[2022-08-07 15:05:35,018] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'load_ad_data', 'load_briefing_data', 'manual__2022-08-07T11:51:07.834855+00:00', '--job-id', '186', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_raw_data_loader.py', '--cfg-path', '/tmp/tmp27q22pd8', '--error-file', '/tmp/tmpd2j9h6c_']
[2022-08-07 15:05:35,018] {standard_task_runner.py:80} INFO - Job 186: Subtask load_briefing_data
[2022-08-07 15:05:35,030] {logging_mixin.py:115} WARNING - /home/hp/miniconda3/envs/checker/lib/python3.8/site-packages/airflow/configuration.py:532 DeprecationWarning: The sql_engine_encoding option in [core] has been moved to the sql_engine_encoding option in [database] - the old setting has been used, but please update your config.
[2022-08-07 15:05:35,043] {task_command.py:371} INFO - Running <TaskInstance: load_ad_data.load_briefing_data manual__2022-08-07T11:51:07.834855+00:00 [running]> on host hp-HP-Pavilion-Laptop-15-eg0xxx
[2022-08-07 15:05:35,070] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=tesfaye
AIRFLOW_CTX_DAG_ID=load_ad_data
AIRFLOW_CTX_TASK_ID=load_briefing_data
AIRFLOW_CTX_EXECUTION_DATE=2022-08-07T11:51:07.834855+00:00
AIRFLOW_CTX_TRY_NUMBER=5
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-07T11:51:07.834855+00:00
[2022-08-07 15:05:35,075] {base.py:68} INFO - Using connection ID 'postgres_connection' for task execution.
[2022-08-07 15:05:35,082] {sql.py:255} INFO - Running statement: COPY briefing FROM './../../data/briefing.csv' WITH DELIMITER AS ';' NULL AS '\null' CSV HEADER;, parameters: None
[2022-08-07 15:05:35,082] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/hp/miniconda3/envs/checker/lib/python3.8/site-packages/airflow/providers/postgres/operators/postgres.py", line 92, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/home/hp/miniconda3/envs/checker/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 234, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/hp/miniconda3/envs/checker/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 260, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedFile: could not open file "./../../data/briefing.csv" for reading: No such file or directory
HINT:  COPY FROM instructs the PostgreSQL server process to read a file. You may want a client-side facility such as psql's \copy.

[2022-08-07 15:05:35,084] {taskinstance.py:1415} INFO - Marking task as UP_FOR_RETRY. dag_id=load_ad_data, task_id=load_briefing_data, execution_date=20220807T115107, start_date=20220807T120534, end_date=20220807T120535
[2022-08-07 15:05:35,092] {standard_task_runner.py:92} ERROR - Failed to execute job 186 for task load_briefing_data (could not open file "./../../data/briefing.csv" for reading: No such file or directory
HINT:  COPY FROM instructs the PostgreSQL server process to read a file. You may want a client-side facility such as psql's \copy.
; 101473)
[2022-08-07 15:05:35,109] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-08-07 15:05:35,122] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
